.github/workflows/build.yml

```diff
@@ -0,0 +1,61 @@
+name: Test
+
+on: 
+  pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Jsonnet
+        run: |
+          mkdir jsonnet
+          curl -o jsonnet.tar.gz -L https://github.com/google/jsonnet/releases/download/v0.14.0/jsonnet-bin-v0.14.0-linux.tar.gz
+          tar -xzf jsonnet.tar.gz -C jsonnet
+          echo "::add-path::jsonnet"
+      - name: Lint jsonnet
+        run: ./scripts/lint_jsonnet.sh
+      - name: Build schemas
+        run: ./scripts/build_schemas.sh
+      - name: Archive schema artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: schemas
+          path: schemas
+  test-schemas:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download built schemas
+        uses: actions/download-artifact@v1
+        with:
+          name: schemas
+      - name: Pull and run validator
+        run: ./scripts/pull_and_run_validator.sh
+      - name: Test schemas
+        run: ./scripts/test_schemas.sh
+  test-translation-templates:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.8'
+      - name: Download built schemas
+        uses: actions/download-artifact@v1
+        with:
+          name: schemas
+      - name: Cache virtualenv
+        uses: actions/cache@v1
+        with:
+            path: ~/.local/share/virtualenvs/
+            key: ${{ runner.os }}-virtualenvs-${{ hashFiles('Pipfile.lock') }}
+      - name: Install Pipenv
+        run: pip install pipenv==2018.11.26
+      - name: Install Virtual Environment
+        run: pipenv install --dev
+      - name: Test Translation Templates
+        run: make test-translation-templates
```

Comment:
We should have another build step that runs `make translate-schemas` to make sure the translation process works.